src/utils/dao-helper.js

```diff
@@ -133,11 +146,52 @@ const convertDatesInObject = (data) => {
   return data;
 };
 
+/**
+ * Converts all keys in an array of objects from snake_case or UPPERCASE to camelCase.
+ *
+ * @param {Array<Object>} rows - The array of objects with keys to be converted.
+ * @returns {Array<Object>} A new array of objects with keys converted to camelCase.
+ */
+const convertKeysToCamelCase = (rows) => rows.map((row) => Object.keys(row).reduce((acc, key) => {
+  const camelCaseKey = key.toLowerCase().replace(/_([a-z])/g, (match, letter) => letter.toUpperCase());
+  acc[camelCaseKey] = row[key];
+  return acc;
+}, {}));
+
+/**
+ * Converts an object's keys from camelCase to snake_case, except for
+ * 'page[number]' and 'page[size]'.
+ *
+ * @param {Object} obj - The object with keys to be converted.
+ * @returns {Object} A new object with keys converted to snake_case.
+ */
+const convertToSnakeCase = (obj) => Object.keys(obj).reduce((acc, key) => {
+  if (key === 'page[number]' || key === 'page[size]') {
+    acc[key] = obj[key];
+  } else {
+    const snakeCaseKey = key.replace(/([A-Z])/g, '_$1').toLowerCase();
+    acc[snakeCaseKey] = obj[key];
+  }
+  return acc;
+}, {});
```

Comment:
Simply use this: https://lodash.com/docs/4.17.15#snakeCase

Reply:
done